src/utils/other.js

```diff
@@ -24,4 +24,32 @@ Logs a value from a template to the console
 
 `;
 
-export const log = value => console.log(value);
\ No newline at end of file
+export const log = value => console.log(value);
+
+export const snapgrid_help = () => `
+
+\`snapgrid(value, gridsize)\`
+
+Snaps object to grid
+
+`;
+
+export const snapgrid = (value, gridsize) => {
```

Comment:
`export const snaptogrid = (value, gridsize) => {`

\+ please replace all occurrences of snapToGrid in codebase (`f-drag`, maybe more)